controls/bsi_app_4_4.yml

```diff
@@ -416,9 +416,58 @@ controls:
       message to the control plane. The control plane SHOULD ONLY accept nodes into a cluster
       that have successfully proven their integrity.

```

Comment:
you should number the sections here.

      (1) Nodes SHOULD send a cryptographically secured (and, if possible, TPM-verified) status
      message to the control plane. (2) The control plane SHOULD ONLY accept nodes into a cluster
      that have successfully proven their integrity.

Reply:
@sluetze Would you like to handle this on a subsequent PR?

Also, the addition of `bsi:` references in the rules is not needed anymore.

Reply:
Both remarks are fixed in the most recent commit